docs/PUBLISHING.md

```diff
@@ -70,16 +71,16 @@ Once merged, wait until the [builds complete on `main`](https://github.com/chanz
 
 #### Publishing the package
 
-7. Pull down the most up-to-date version of main: `git checkout main && git pull && yarn build`
-8. Publish the package: `npm publish`
-9. Create a [new release](https://github.com/chanzuckerberg/edu-design-system/releases) based on the new tag. Use the same text used for the pull request description above (from CHANGELOG.md). Also include the link for the built storybook in the description. This will automatically post to [relevant slack channels](https://slack.github.com/):
-10. Lastly, run the following to "back merge" release changes to `next`:
+8. Pull down the most up-to-date version of main: `git checkout main && git pull && yarn build`
```

Comment:
Can we also have `yarn install` in this chain of commands to use the proper deps before building?

Reply:
Good idea. Should add to both cases for extra safety